code/drasil-example/Drasil/NoPCM/ODEs.hs

```diff
@@ -0,0 +1,19 @@
+module Drasil.NoPCM.ODEs (noPCMODEOpts, noPCMODEInfo) where
+
+import Language.Drasil.Code (odeInfo, odeOptions, quantvar, ODEInfo,
+  ODEMethod(RK45), ODEOptions)
+import Language.Drasil.CodeExpr
+
+import Data.Drasil.Quantities.Physics (time)
+
+import Drasil.SWHS.Unitals (tauW, tempC, tempInit, tempW, timeFinal, timeStep,
+  absTol, relTol)
+
+
+noPCMODEOpts :: ODEOptions
+noPCMODEOpts = odeOptions RK45 (sy absTol) (sy relTol) (sy timeStep) (exactDbl 0)
+
+noPCMODEInfo :: ODEInfo
+noPCMODEInfo = odeInfo (quantvar time) (quantvar tempW)
+  [quantvar tauW, quantvar tempC] (exactDbl 0) (sy timeFinal) (sy tempInit)
+  [recip_ (sy tauW) `mulRe` (sy tempC $- idx (sy tempW) (int 0))] noPCMODEOpts
```

Comment:
I forgot to mention this -- I moved the ODEs to a new file so that we could create `CodeExpr`s instead of `Expr`s by importing `Language.Drasil.CodeExpr` instead of `Language.Drasil` -- it has the same function names and operators, so it looks the same but creates different things (CodeExpr vs Expr). If we wanted to keep it in the original Body.hs file, we'd probably need to import the CodeExpr with a qualified name, which would also make the equations a bit more difficult to read.

Reply:
Saw that - good move.